src/containers/VisualElement/VisualElementSearch.jsx

```diff
@@ -48,8 +48,18 @@ const VisualElementSearch = ({
                 resource_id: image.id,
                 size: 'fullbredde',
                 align: '',
-                alt: alttextsI18N(image, locale, true) || '',
-                caption: captionsI18N(image, locale, true) || '',
+                alt: convertFieldWithFallback(
+                  image.alttext,
+                  'alttext',
+                  image.alttext.alttext,
+                  locale,
+                ),
+                caption: convertFieldWithFallback(
+                  image.caption,
+                  'caption',
+                  image.caption.caption,
```

Comment:
? 

Du burde ikke gjøre det slik. Da faller poenget med denne funksjonen bort.  Hvis du ser i convertFieldWithFallback så trenger du ikke ha med lokale for å få denne funksjonaliteten.